cpp/include/cudf/table/table.hpp

```diff
@@ -112,6 +112,10 @@ class table {
    *---------------------------------------------------------------------------**/
   table_view select(std::vector<cudf::size_type> const& column_indices) const;
 
+  column& get_column(cudf::size_type i) { return *(_columns.at(i)); }
```

Comment:
Doc. (Make sure to document that this can throw `std::out_of_range` if `i` is out of bounds.